smoke-test/spark-smoke-test/smoke.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+
+# Script assumptions:
+#   - The gradle build has already been run.
+#   - Python 3.6+ is installed and in the PATH.
+#   - pytest is installed
+#   - requests is installed
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+cd "$DIR"
+
+python3 -m venv venv
+source venv/bin/activate
+pip install --upgrade pip wheel setuptools
+pip install -r requirements.txt
+
+echo "--------------------------------------------------------------------"
+echo "Setting up datahub server"
+echo "--------------------------------------------------------------------"
+
+
+datahub docker quickstart \
```

Comment:
why not use the python way of starting the docker containers (like the other smoke tests)